porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
Here's a specific example of when tagged != deployed, if it's helpful to anyone:

For cf.gov, our WordPress theme and plugin deployments are controlled by tag number. You can tag new releases all day long, but until you update them in [these](https://github.cfpb.gov/CFPB/consumerfinance.gov/blob/master/deployment_config/build/wordpress.yaml) [deployment](https://github.cfpb.gov/CFPB/consumerfinance.gov/blob/master/deployment_config/content/wordpress.yaml) [files](https://github.cfpb.gov/CFPB/consumerfinance.gov/blob/master/deployment_config/prod/wordpress.yaml) and a deployment is run, they are not deployed.